src/taipy/core/_repository/_sql_model.py

```diff
@@ -42,7 +43,20 @@ class _TaipyVersion(Base):  # type: ignore
 
     id = Column(Text, primary_key=True)
     config = Column(Text)
-
-    def __int__(self, id: str, config: Json):
+    creation_date = Column(Text)
+    is_production = Column(Boolean)
+    is_development = Column(Boolean)
+    is_latest = Column(Boolean)
```

Comment:
This is not consistent with the current FS repo.

Can we have a separate table to store which one is production, which one is latest version, ... ?

Reply:
Yes, it's not consistent and I don't think we should mimic the FS implementation. For this is the correct data model considering a SQL database. Same for mongo, I would rather that each version document would keep track of its status, be it latest, development or production. Adding another table over complicates things and don't take advantage of database capabilities. I understand your implementation on the FS, but I don't think it should be the model for the others.